README.md

```diff
@@ -37,7 +37,7 @@ runners are supported:
 | **Racket**     | RackUnit                                                                                    | `rackunit`                                                                                                      |
 | **Ruby**       | Cucumber, [M], [Minitest][minitest], Rails, RSpec                                           | `cucumber`, `m`, `minitest`, `rails`, `rspec`                                                                   |
 | **Rust**       | Cargo                                                                                       | `cargotest`                                                                                                     |
-| **Scala**      | SBT                                                                                         | `sbttest`                                                                                                       |
+| **Scala**      | SBT,                                                                                        | `sbttest`                                                                                                       |
```

Comment:
It seems you've started adding the new runner here, but it somehow didn't get through all the way.

Reply:
oops, seems like it's lost somewhere. Fixed this! Thanks for reviewing :D